ErikBagratyan/chapter_7_ problem_ 8

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+#include <vector>
+#include <ctime>
+
+void show_reverse_vector(const std::vector<int>& vec, int vec_size)
+{
+    if(vec_size == 0) return;
+    std::cout << vec[vec_size - 1] << " ";
+    
+    return show_reverse_vector(vec,--vec_size);
```

Comment:
Useful to know, you could also use pop_back function for vector. https://www.cplusplus.com/reference/vector/vector/pop_back/